imgui-SFML.cpp

```diff
@@ -558,8 +559,9 @@ void Image(const sf::Texture& texture, const sf::Vector2f& size,
            const sf::Color& tintColor, const sf::Color& borderColor) {
     ImTextureID textureID =
         convertGLTextureHandleToImTextureID(texture.getNativeHandle());
-    ImGui::Image(textureID, size, ImVec2(0, 0), ImVec2(1, 1), tintColor,
-                 borderColor);
+    
+    ImGui::Image(textureID, ImVec2(size.x,size.y), ImVec2(0, 0), ImVec2(1, 1), ImColor( tintColor.toInteger()),
```

Comment:
Please make helper function in anonymous namespace in this file and use it instead:

```cpp
ImColor toImColor(sf::Color c) {
    return ImColor(static_cast<int>(c.r), static_cast<int>(c.g), static_cast<int>(c.b), static_cast<int>(c.a));
}
```